packages/fast-components-styles-msft/src/utilities/color/neutral-layer.ts

```diff
@@ -44,45 +57,64 @@ const darkNeutralLayerL4: DesignSystemResolver<number> = (
     return darkRefIndex;
 };
 
-export const neutralLayerL1: SwatchResolver = swatchByMode(PaletteType.neutral)(
+/**
+ * Used as the background color for floating layers like context menus and flyouts.
+ */
+export const neutralLayerFloating: SwatchResolver = swatchByMode(PaletteType.neutral)(
     0,
-    subtract(darkNeutralLayerL4, multiply(neutralFillCardDelta, 3))
+    subtract(darkNeutralLayerL4, multiply(neutralFillCardDelta, 5))
```

Comment:
This is going to cause visual changes to designs. Please ping channels to notify folks as this goes in, especially since this isn’t technically breaking but it *could* appear as a visual regression to them.

Reply:
The changes don’t quite line up correctly here, but I’m posting this update now.